tests/NATS.Client.Perf/Program.cs

```diff
@@ -11,7 +11,7 @@
     Subject = "test",
     PubTasks = 10,
     MaxNatsBenchRatio = 0.20,
-    MaxMemoryMb = 500,
+    MaxMemoryMb = 600,
```

Comment:
Not sure why it's this high on github instances. On my local Linux VM it's about 150M.

Reply:
Hmm, mine is around 325M on a pretty fast Linux Desktop.  Probably something about # of Cores (is your VM 1-core?) plus the slow GitHub instance CPUs causing buffers to build up